admin_ui_support/src/Plugin/rest/resource/PermissionsCollectionResource.php

```diff
@@ -105,7 +105,7 @@ public function get() {
       $permissions = $this->permissionHandler->getPermissions();
       foreach ($permissions as $id => $permission) {
         $permissions[$id]['id'] = $id;
-        $permissions[$id]['provider_label'] = $this->moduleExtensionList->getName($permissions[$id]['provider']);
+        $permissions[$id]['provider_label'] = $this->moduleHandler->getName($permissions[$id]['provider']);
```

Comment:
Why do have these changes? I think this is wrong given getName is not meant to be the actual API.

Reply:
@tedbow

Reply:
Yeah I think I was trying to get this to work with 8.5.x  since the 'extension.list.module' service doesn't exist in this branch.

Not sure if we care about 8.5.x core for this work.